build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Install gzip, snappy and thrift >= 0.11.0
+
```

Comment:
Hint: I know it's just a temporary script, but you can simply call
```bash
set -e
```
here, and drop all the `&&`'s after each line. From bash manual:
```
  -e  Exit immediately if a command exits with a non-zero status.
```